docs/development/introducingchartlibrary.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: "Introducing New Chart Library"
+description: "Introducing New Chart Library"
+group: development
+---
+{% include JB/setup %}
+
+### Why Charts are important in zeppeline?
+Zeppeline is mostly used for data analysis and visualization. Depending on the user requirements and datasets the types of charts needed could differ. So Zeppeline let user to add different chart libraries and chart types.
+
+<br />
+### Add New Chart Library
+When needed a new JS chart library than D3 (nvd3) which is included in zeppelin, a new JS library for zeppelin-web is added by adding name in zeppelin-web/bower.json
+
+eg: Adding map visualization to Zeppeline using leaflet
```

Comment:
Thanks, Felix

for pointing out.

On Wed, Aug 12, 2015 at 2:28 AM, Felix Cheung <notifications@github.com>
wrote:

> In docs/development/introducingchartlibrary.md
> <https://github.com/apache/incubator-zeppelin/pull/203#discussion_r36798048>
> :
>
> > +---
> > +layout: page
> > +title: "Introducing New Chart Library"
> > +description: "Introducing New Chart Library"
> > +group: development
> > +---
> > +{% include JB/setup %}
> > +
> > +### Why Charts are important in zeppeline?
> > +Zeppeline is mostly used for data analysis and visualization. Depending on the user requirements and datasets the types of charts needed could differ. So Zeppeline let user to add different chart libraries and chart types.
> > +
> > +<br />
> > +### Add New Chart Library
> > +When needed a new JS chart library than D3 (nvd3) which is included in zeppelin, a new JS library for zeppelin-web is added by adding name in zeppelin-web/bower.json
> > +
> > +eg: Adding map visualization to Zeppeline using leaflet
>
> same here. multiple in this doc
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-zeppelin/pull/203/files#r36798048>.
>



-- 
Cheers,
Madhuka Udantha
http://madhukaudantha.blogspot.com